### PR TITLE
[DONE and TESTED] Add Workvisa/Passports/Passcards to the game and makes all accessories point free.

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
<details>
<summary>
	About The Pull Request:

Sprites credits for MOST parts to: 
https://github.com/Aurorastation/Aurora.3

I ported a few sprites from aurora like passports so people can RP with it and also made all accessories free because needing to spend 8/10 of all of your points to get fluffy items that are harmless seems a little keke and bad.

image

	
<hr>
</details>

## Changelog
:cl:
add: Passports/Workvisa/Passcards into the game on the accessories menu
balance: made all accessories free point-wise.

/:cl:

### CODE REVIEW

**Comment by TheSleepyCatmom** [comment]
For the most part, Nadezhda would not issue any specific passcards, passports, or visas to anyone. Period. The only exception I can see to this being Kriosan citizens because they're the only legitimate entity in nearby space recognized by the High Council as an ally that has a border as a country and isn't essentially a roaving band of.. well, pirates.

For the sake of this, they would not be issued passcards from Sol. Ever. Why would you issue someone passcards from Sol when most of the Colony thinks Sol are massive terrorist fuckwits and left to avoid them? It makes no sense. Keeping the ones for Nadezhda like Work Visas and stuff is fine, I suppose, but I'm not for having Solarian backgrounds issued into the colony in such a way. People openly toting these things around would be jumped by half the population, we are *rebels* who openly oppose Solarian rule. (Plus Kurilskaya is hidden and a criminal underworld, they wouldn't issue you a passcard anyway.)

**Comment by Guidesu** [comment]
> For the most part, Nadezhda would not issue any specific passcards, passports, or visas to anyone. Period. The only exception I can see to this being Kriosan citizens because they're the only legitimate entity in nearby space recognized by the High Council as an ally that has a border as a country and isn't essentially a roaving band of.. well, pirates.
> 
> For the sake of this, they would not be issued passcards from Sol. Ever. Why would you issue someone passcards from Sol when most of the Colony thinks Sol are massive terrorist fuckwits and left to avoid them? It makes no sense. Keeping the ones for Nadezhda like Work Visas and stuff is fine, I suppose, but I'm not for having Solarian backgrounds issued into the colony in such a way. People openly toting these things around would be jumped by half the population, we are _rebels_ who openly oppose Solarian rule. (Plus Kurilskaya is hidden and a criminal underworld, they wouldn't issue you a passcard anyway.)

Sigh.....
Removed the sol passcard and let the sol passport just as fluffy if someone has a sol character that somehow and for some reason stored their old and unvalid passport.

 Nadezhda would not issue any specific passcards, passports, or visas to anyone.... alright so now the desc are changed to reflect that said person brought their old and unvalid visas to this new colony

**Comment by Guidesu** [comment]
Work Visa and Kriosan passport are still in and anymore feedback is alright

File: code/modules/clothing/accessories/accessory.dm [hunk 1]
```diff
@@ -1175,3 +1175,107 @@
...
+
+/obj/item/clothing/accessory/passcard/rankorin
+	name = "Rankorin passcard"
+	desc = "A passcard issued to citizens of Rankorin, a death world due to its highly toxic atmosphere caused by a combination of natural flora and common storms of poisonous dusts clouds."
+	icon_state = "passcard_rankorin"
+	item_state = "pass"
+
+/obj/item/clothing/accessory/passcard/mardinat_yunan
+	name = "Mardinat Yanun passcard"
+	desc = "A passcard issued to citizens of the Mardinat Yanun, initially selected for habitation for its rich gasses and potential for being located on an upcoming trade lane, this gas giant had many of its economic dreams dashed, with the gaseous mixture of the atmosphere being poor for fuel production and the trade lane never materializing. You wonder how this passcard was even issued due to the failure of the colony itself."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1224]:
```suggestion
	desc = "A passcard issued to citizens of Mardinat Yunan, initially selected for habitation for its rich gasses and potential for being located on an upcoming trade lane. This gas giant had many of its economic dreams dashed, with the gaseous mixture of the atmosphere being poor for fuel production and the trade lane never materializing."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 2]
```diff
@@ -1175,3 +1175,107 @@
...
+	icon_state = "passcard_madinat_yunan"
+	item_state = "badge"
+
+/obj/item/clothing/accessory/passcard/thalatta
+	name = "Thalatta passcard"
+	desc = "A passcard issued to the crewmembers of the Thalatta ship, translated from the ancient Greek term for 'The Sea' , Thalatta is an aptly named city-ship that is ever sailing through the great cosmic sea."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1230]:
```suggestion
	desc = "A passcard issued to crewmembers of the City-Ship Thalatta, translated from the ancient Greek term for 'The Sea.' Thalatta is an aptly named mobile city that is ever-sailing through the great cosmic sea."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 3]
```diff
@@ -1175,3 +1175,107 @@
...
+	icon_state = "passcard_thalatta"
+	item_state = "pass"
+
+/obj/item/clothing/accessory/passcard/wanderers_armada
+	name = "Wanderers Armada passcard"
+	desc = "A passcard issued to crewmembers of who are from the fleet ship's of Wanderers Armada. The Wanderer's Armada is a group of nearly a hundred repurposed ships, ranging from small corvettes to the largest Dreadnought that began the Armada. While the size of this fleet rivals certain Federation fleets, they visit each system and recruiting those that wish to join them."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1236]:
```suggestion
	desc = "A passcard issued to the crew and passengers of the Wanderer's Armada. The Wanderer's Armada is a flotilla of repurposed ships, ranging from small corvettes to the largest Dreadnought that began the Armada. While the size of this fleet rivals certain Federation fleets, they visit each system and recruit those that wish to join them."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 4]
```diff
@@ -1175,3 +1175,107 @@
...
+	icon_state = "passcard_wanderers_armada"
+	item_state = "badge"
+
+/obj/item/clothing/accessory/passcard/onkarth
+	name = "Onkarth passcard"
+	desc = "A passcard issued to citizens of the Onkarth, the homeworld of the sablekyne, a desert planet of extreme conditions along the equator and northern fringes of civilization."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1242]:
```suggestion
	desc = "A passcard issued to citizens of Onkarth, the homeworld of the Sablekyne, a primarily desert planet of extreme conditions along the equator and northern fringes of civilization."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 5]
```diff
@@ -1175,3 +1175,107 @@
...
+	icon_state = "passcard_onkarth"
+	item_state = "passport"
+
+/obj/item/clothing/accessory/passcard/neapolis
+	name = "Neapolis passcard"
+	desc = "A passcard issued to citizens of the Neapolis, an lush garden world has modeled itself as an escape for repressed artists, political exiles, and individuals who would largely be ostracized by most major powers."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1248]:
```suggestion
	desc = "A passcard issued to citizens of Neapolis, a lush garden world that has modeled itself as an escape for repressed artists, political exiles, and individuals who would largely be ostracized by most major powers."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 6]
```diff
@@ -1175,3 +1175,107 @@
...
+	icon_state = "passcard_neopolis"
+	item_state = "passport"
+
+
+/obj/item/clothing/accessory/passcard/aqua_fria
+	name = "Aqua Fria passcard"
+	desc = "A passcard issued to citizens of the Aqua Fria, tucked away in a heavily populated system, this large aquatic world has made a reputation for being a substantial food source and home to a number of research institutes."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1255]:
```suggestion
	desc = "A passcard issued to citizens of Aqua Fria. Tucked away in a heavily populated system, this large aquatic world has made a reputation for being a substantial food source and home to a number of research institutes."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 7]
```diff
@@ -1175,3 +1175,107 @@
...
+	icon_state = "passcard_aqua_fria"
+	item_state = "badge"
+
+/obj/item/clothing/accessory/passcard/donbettyr
+	name = "Donbettyr passcard"
+	desc = "A passcard issued to citizens of the Donbettyr, the homeworld of the akula, an old primarily water planet with two moons and few thousand separated areas of land making up a series of islands and archipelagos."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1261]:
```suggestion
	desc = "A passcard issued to citizens of Donbettyr, the homeworld of the Akula. An old, primarily ocean world with two moons and few thousand separated areas of land making up a series of islands and archipelagos."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 8]
```diff
@@ -1175,3 +1175,107 @@
...
+	icon_state = "passcard_aqua_fria"
+	item_state = "badge"
+
+/obj/item/clothing/accessory/passcard/passport_sol
+	name = "old solarian passport"
+	desc = "A old passport issued to citizens of the sol sector.... why is it here? Is it a trinket?"
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1267]:
```suggestion
	desc = "An old passport issued to citizens of the Solarian Federation, this one seems to have been issued before the Bluespace Collapse. Why is it here? Is it a trinket?"
```

File: code/modules/clothing/accessories/accessory.dm [hunk 9]
```diff
@@ -1175,3 +1175,107 @@
...
+	icon_state = "passport_sol"
+	item_state = "badge"
+
+/obj/item/clothing/accessory/passcard/passport_general
+	name = "passport"
+	desc = "A passport issued to citizens of the Nadezhda that live in the local sector in the Chromin 8 system, Amethyn."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1273]:
```suggestion
	desc = "A passport issued to Nadezhda Colonists that live on Amethyn, in the Chromin 8 system."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 10]
```diff
@@ -1175,3 +1175,107 @@
...
+	icon_state = "general_passport"
+	item_state = "general_passport"
+
+/obj/item/clothing/accessory/passcard/passport_kriosan
+	name = "Kriosan passport"
+	desc = "A passport issued to citizens of the Krios that are working or living in the local sector temporarily or permanetly in the Chromin 8 system, Amethyn."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1279]:
```suggestion
	desc = "A passport issued to Kriosans by the Kriosan Government who are working or living on Amethyn, in the Chromin 8 system."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 11]
```diff
@@ -1175,3 +1175,107 @@
...
+
+/obj/item/clothing/accessory/passcard/sanperitor
+	name = "Sanperitor passcard"
+	desc = "A passcard issued to citizens of Sanperitor, an low atmosphere world closer to its systems star then most species are comfortable living with. The intense solar radiation of the planet has caused those who reside their to naturally adapt to the toxins present in every day life"
+	icon_state = "passcard_sanperitor"
+	item_state = "pass"
+
+/obj/item/clothing/accessory/passcard/rankorin
+	name = "Rankorin passcard"
+	desc = "A passcard issued to citizens of Rankorin, a death world due to its highly toxic atmosphere caused by a combination of natural flora and common storms of poisonous dusts clouds."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1218]:
```suggestion
	desc = "A passcard issued to citizens of Rankorin. Rankorin is in many parts considered a 'death world' due to its highly toxic atmosphere, caused by a combination of natural flora and common storms of poisonous dusts clouds."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 12]
```diff
@@ -1175,3 +1175,107 @@
...
+
+/obj/item/clothing/accessory/passcard/coalition_of_colonies
+	name = "Coalition passcard"
+	desc = "A passcard issued to citizens of the unknown exo-planets away from Sol space or Chromin itself. It generally means that they are from a coalition of independents colonies that are not allied to sol nor Nadezhda."
+	icon_state = "passcard_coc"
+	item_state = "badge"
+
+/obj/item/clothing/accessory/passcard/sanperitor
+	name = "Sanperitor passcard"
+	desc = "A passcard issued to citizens of Sanperitor, an low atmosphere world closer to its systems star then most species are comfortable living with. The intense solar radiation of the planet has caused those who reside their to naturally adapt to the toxins present in every day life"
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1212]:
```suggestion
	desc = "A passcard issued to citizens of Sanperitor, an low atmosphere world closer to its system's star then most species are comfortable living with. The intense solar radiation of the planet has caused those who reside there to naturally adapt to the toxins present in everyday life."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 13]
```diff
@@ -1175,3 +1175,107 @@
...
+	icon_state = "workvisa"
+	item_state = "pass"
+
+/obj/item/clothing/accessory/passcard/new_damascus
+	name = "New Damascus passcard"
+	desc = "A passcard card issued to citizens of New Damascus, once a bustling manufacturing world that was rich in oxygen and low in moisture, New Damascus is an arid world slowly bringing itself back from the brink of economic collapse."
+	icon_state = "passcard_new_damascus"
+	item_state = "passport"
+
+/obj/item/clothing/accessory/passcard/coalition_of_colonies
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1204]:
```suggestion
/obj/item/clothing/accessory/passcard/fringe_colonies
```

File: code/modules/clothing/accessories/accessory.dm [hunk 14]
```diff
@@ -1175,3 +1175,107 @@
...
+	name = "Coalition passcard"
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1205]:
```suggestion
	name = "Outer Fringe passcard"
```

File: code/modules/clothing/accessories/accessory.dm [hunk 15]
```diff
@@ -1175,3 +1175,107 @@
...
+	desc = "A passcard issued to citizens of the unknown exo-planets away from Sol space or Chromin itself. It generally means that they are from a coalition of independents colonies that are not allied to sol nor Nadezhda."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1206]:
```suggestion
	desc = "A passcard issued to citizens and colonists away from Sol space or Chromin itself. The look of this passcard may vary group to group, but no matter where it comes from, it's generally understood that whoever owns this passcard has never been a Solarian citizen."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 16]
```diff
@@ -1175,3 +1175,107 @@
...
+
+/obj/item/clothing/accessory/passcard/work_visa
+	name = "work visa"
+	desc = "A working card issued to personal of the Nadezhda that are working in the local sector temporarily in the Chromin 8 system, Amethyn. "
+	icon_state = "workvisa"
+	item_state = "pass"
+
+/obj/item/clothing/accessory/passcard/new_damascus
+	name = "New Damascus passcard"
+	desc = "A passcard card issued to citizens of New Damascus, once a bustling manufacturing world that was rich in oxygen and low in moisture, New Damascus is an arid world slowly bringing itself back from the brink of economic collapse."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1200]:
```suggestion
	desc = "A passcard card issued to citizens of New Damascus, once a bustling manufacturing world that was rich in oxygen and low in moisture. New Damascus is an arid world slowly bringing itself back from the brink of economic collapse."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 17]
```diff
@@ -1175,3 +1175,107 @@
...
+
+/obj/item/clothing/accessory/passcard/resident_card
+	name = "resident passcard"
+	desc = "A resident card issued to citizens of the Nadezhda in the Chromin 8 system, Amethyn."
+	icon_state = "resident_card"
+	item_state = "passport"
+
+/obj/item/clothing/accessory/passcard/work_visa
+	name = "work visa"
+	desc = "A working card issued to personal of the Nadezhda that are working in the local sector temporarily in the Chromin 8 system, Amethyn. "
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1194]:
```suggestion
	desc = "A card issued by the Nadezhda Colony for colonists who are offered residency for the purpose of work on Amethyn, in the Chromin 8 system. "
```

File: code/modules/clothing/accessories/accessory.dm [hunk 18]
```diff
@@ -1175,3 +1175,107 @@
...
+
+/obj/item/clothing/accessory/passcard
+	name = "general passcard"
+	desc = "A passcard issued to citizens of the Nadezhda in the Chromin 8 system, Amethyn."
+	icon_state = "passcard_general"
+	item_state = "general_passport"
+
+/obj/item/clothing/accessory/passcard/resident_card
+	name = "resident passcard"
+	desc = "A resident card issued to citizens of the Nadezhda in the Chromin 8 system, Amethyn."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1188]:
```suggestion
	desc = "A card issued by the Nadezhda Colony for colonists who are offered temporary residency on Amethyn, in the Chromin 8 system."
```

File: code/modules/clothing/accessories/accessory.dm [hunk 19]
```diff
@@ -1175,3 +1175,107 @@
 	item_state = "sweatervest_red"
 	overlay_state = "sweatervest_red"
 
+/* Passports/work visa/passcard */
+
+/obj/item/clothing/accessory/passcard
+	name = "general passcard"
+	desc = "A passcard issued to citizens of the Nadezhda in the Chromin 8 system, Amethyn."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/accessories/accessory.dm:1182]:
```suggestion
	desc = "A passcard issued by the Nadezhda Colony when someone gains permanent residency on Amethyn, in the Chromin 8 system."
```

**Review by TheSleepyCatmom — COMMENTED** [review]
Grammatical modifications, shuffling, clarifications, and making things flow better.

**Comment by Guidesu** [comment]
Taking a break from sojourn

**Comment by Guidesu** [comment]
boop

**Comment by cdb-is-not-good** [comment]
I'll need to look this over when the weekend is done. Has it been adjusted with the requested changes?

**Comment by Stutternov** [comment]
Overall - I feel this is a good PR as it is mostly flavor. 

The loadout items are reasonable after feedback and allow for players to easier-roleplay being 'new characters' or sporting flavor of the colony being a sort of 'colony of rejects' / 'colony of refugees' or so forth. 

Some of these may also be useful in events if we attempt to make more special characters from the listed regions or use a sprite as a base and rename where the passport is from. Nice touch of flavor.

**Comment by cdb-is-not-good** [comment]
Took a look and it seems fine other than the one for the Wanders armadda- the flotilla as far as I'm aware isn't so cohesive as to have something like a passcard for individual crew members of each ship

**Pull request merged**